templates/api-template-nodejs/package.json

```diff
@@ -9,8 +9,7 @@
     "test": "mocha"
   },
   "dependencies": {
-    "request": "*",
-    "deasync": "*"
+    "request": "*"
```

Comment:
We wouldn't need it in the template. Let's remove.

Reply:
updated